jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/SliderRange.java

```diff
@@ -84,50 +86,53 @@ public void setValue(int thumbIndex, int value) {
     int width = Integer.parseInt(styles[1]);
     String newStyle = setNewStyle(trackStyle, thumbIndex, value, left, width);
 
-    reflectionSetValue(thumbIndex, newThumbStyle, newStyle, trackStyle,value, left, width);
+    reflectionSetValue(thumbIndex, newThumbStyle, newStyle, value, left, width);
 
   }
 
 
-  private boolean isSwitch(String style, int thumbIndex, int value, int left, int width) {
+  private boolean isSwitch(int thumbIndex, int value, int Left, int width) {
+    int localLeft = Left;
+
     switch (thumbIndex) {
-      case 1: return  (value > width + left);
-      case 2: return (value < left);
+      case 1: return  (value > width + localLeft);
+      case 2: return (value < localLeft);
+      default: break;
     }
     return false;
   }
-
-  private String setNewStyle(String style, int thumbIndex, int value, int left, int width) {
-
+  private String setNewStyle(String style, int thumbIndex, int value, int Left, int width) {
```

Comment:
Please use "left" name of the variable starting from small letter

Reply:
renamed